lib/src/posthog_flutter_io.dart

```diff
@@ -413,6 +420,34 @@ class PosthogFlutterIO extends PosthogFlutterPlatformInterface {
     }
   }
 
+  @override
+  Future<void> captureException({
+    required Object error,
+    StackTrace? stackTrace,
+    Map<String, Object>? properties,
+    bool handled = true,
+  }) async {
+    if (!isSupportedPlatform()) {
+      return;
+    }
+
+    try {
+      final exceptionData = DartExceptionProcessor.processException(
+        error: error,
+        stackTrace: stackTrace,
+        properties: properties,
+        handled: handled,
+        inAppIncludes: _config?.errorTrackingConfig.inAppIncludes,
+        inAppExcludes: _config?.errorTrackingConfig.inAppExcludes,
+        inAppByDefault: _config?.errorTrackingConfig.inAppByDefault ?? true,
+      );
+
+      await _methodChannel.invokeMethod('captureException', exceptionData);
```

Comment:
before we call the method, we need to be sure that `exceptionData` is serializable, since `properties` is `Map<String, Object>`, we cannot trust the values, we do something similar for Android and iOS, we'd need to normalize the data here too otherwise this method will fail